### PR TITLE
[tests-only][full-CI] adding test for updating share link of a resource in project space.

### PR DESCRIPTION
**Pull request by prashant-gurung899**
## Description
This PR add tests coverage for share-ng endpoint to update share link of a resource in project-space
```feature
Scenario Outline: update role of a folder's link share inside project-space using permissions endpoint
Scenario Outline: update role of a file's link share inside a project-space using permissions endpoint
Scenario Outline: update role of a file's link share to internal inside a project-space using permissions endpoint
Scenario Outline: update role of a folder's link share to internal inside project-space using permissions endpoint
```
## Related Issue

- Related to : https://github.com/owncloud/ocis/issues/9046

## How Has This Been Tested?

- locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature [hunk 1]
```diff
@@ -2886,4 +2886,75 @@ Feature: Create a share link for a resource
       | createOnly       | view                 |
       | createOnly       | edit                 |
       | createOnly       | upload               |
-      | blocksDownload   | blocksDownload       |
\ No newline at end of file
+      | blocksDownload   | blocksDownload       |
+
+
+  Scenario Outline: update link share of a file inside a project-space
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature:2892]:
```suggestion
  Scenario Outline: update link share of a file inside a project-space (permissions endpoint)
```

Comment (prashant-gurung899) [reply tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature:2892]: @saw-jan  should I change it to : 
update link share of a file inside a project-space using permissions endpoint 
OR
update link share of a file inside a project-space (permissions endpoint)

File: tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature [hunk 2]
```diff
@@ -2971,3 +2971,75 @@ Feature: Create a share link for a resource
       | createOnly       | view                 |
       | createOnly       | edit                 |
       | createOnly       | upload               |
+
+
+
```
Comment (S-Panta) [inline tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature:2976]:
```suggestion


```

File: tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature [hunk 3]
```diff
@@ -2971,3 +2971,74 @@ Feature: Create a share link for a resource
       | createOnly       | view                 |
       | createOnly       | edit                 |
       | createOnly       | upload               |
+
+
+  Scenario Outline: update link share of a file inside a project-space using permissions endpoint
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature:2976]: Please specify what updates you are making to your scenario Outline

File: tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature [hunk 4]
```diff
@@ -2971,3 +2971,74 @@ Feature: Create a share link for a resource
...
+                }
+              }
+            }
+          }
+        }
+        """
+    Examples:
+      | permissions-role | new-permissions-role |
+      | view             | edit                 |
+      | edit             | view                 |
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature:3044]: any specific reason for limiting permissions-role to vie and edit only?

**Review by amrita-shrestha — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature (hunk 4 above)
Comment (prashant-gurung899) [reply tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature:3044]: It was due to different JSON response for edit/view and internal permissions role. I have added new scenario for updating role of a file to internal.

**Review by amrita-shrestha — APPROVED** [review]
lgtm

**Review by saw-jan — APPROVED** [review]

**Pull request merged**